routing_model/src/actions.rs

```diff
@@ -197,6 +197,7 @@ impl Action {
             let mut sorted_values = our_current_nodes
                 .values()
                 .cloned()
+                .filter(|node_state| !node_state.is_offline)
```

Comment:
I think we probably more want to adjust the sorting the same way it is done for relocating a few lines bellow.